docs/firstobject.md

```diff
@@ -76,78 +76,84 @@ There is of course a lot more to Nodes, Content Types, Formatters, Widgets and i
 </div>
 </details>
 
-![Add Content](../imgs/Add-content.jpg)
+# Adding Content
 
-Here you see all the `Content Types` defined by default in Archipelago. Yes, you got it right, go on and click on `Digital Object` to get your first Digital Object Node.
+Below you see all the `Content Types` defined by default in Archipelago. Let's click on `Digital Object` to get your first Digital Object Node.
```

Comment:
Again about the visual 'location' of things. If we have a screenshot, with `below` you mean where? Thoughts for beta3